app/controllers/api/v1/study_files_controller.rb

```diff
@@ -750,7 +750,10 @@ def study_file_params
           ann_data_file_info_attributes: [
             :_id, :reference_file, :has_clusters, :has_metadata, :has_expression, :has_raw_counts,
             data_fragments: AnnDataFileInfo::DATA_FRAGMENT_PARAMS
-          ]
+          ],
+          differential_expression_file_info_attributes: [
+            :_id, clustering_association: [], annotation_association: []
```

Comment:
Non-blocking: I know that the Rails-integration part is scoped in SCP-5096, so this can be addressed there. 
 That being said, if we are adding these parameters here, then they should be defined in `differential_expression_file_info.rb`.  Also, according to SCP-4999, our current assumption is one annotation per file, which would also mean a single cluster as well.  In that case, they can be strings rather than arrays.  We only need arrays if there's a one-to-many relationship.  Also, I would favor naming the first one `cluster_group_id`.

Reply:
Switched to strings and added the fields to `differential_expression_file_info.rb`
 I liked the naming to all match i.e. all ending with `_association` so I did not change the `clustering_assocation` to `cluster_group_id` but I'm not so married to it that if it gets changed later (like in [SCP-5096](https://broadworkbench.atlassian.net/browse/SCP-5096)) that's no problem